docs/pages/kubernetes-access/getting-started/agent.mdx

```diff
@@ -1,10 +1,10 @@
 ---
-title: Connect a Kubernetes Cluster to Teleport
-description: Connecting a Kubernetes cluster to Teleport
+title: Connect Kubernetes Cluster to Teleport
+description: Connecting Kubernetes cluster to Teleport
```

Comment:
Why remove `a`? I think it read better before.

Reply:
restored, this got mixed up in the copying.